cms/templates/product_page.html

```diff
@@ -33,10 +34,38 @@
             {% endif %}
               <h1>{{ page.title }}</h1>
               {# Description field contents are already rendered wrapped in a <p> tag #}
-              
+
               <section class="course-description">
                 {{ page.description | richtext }}
               </section>
+        </div>
+        </div>
+          <div class="col order-md-1 order-2 flex-grow-1" id="product-details">
```

Comment:
This seems to throw the info box to the left on my system (Firefox 119 on macOS Sonoma in the screenshot but also in Chrome 119):

![image](https://github.com/mitodl/mitxonline/assets/945611/2986b8a8-4468-4322-956c-3fd72678cfa4)

(FWIW, I was able to fix this in the inspector by adding `order-sm-1 order-md-2` to the infobox column but that broke in the mobile view.)